src/script/entity/Conversation.ts

```diff
@@ -82,6 +82,10 @@ enum TIMESTAMP_TYPE {
   MUTED = 'mutedTimestamp',
 }
 
+export const isLastReceivedMessage = (messageEntity: Message, conversationEntity: Conversation): boolean => {
+  return messageEntity.timestamp() >= conversationEntity.last_event_timestamp();
+};
```

Comment:
because in this case, I need to identify if the given (rendered) message is the last message, to add to it `onVisible` and `onVisibilityLost`. but in many cases it can be replaced with `hasLastReceivedMessageLoaded` - I'm planning to do the refactoring of it later, within the bug that I discovered. here I just wanted to move it out of React component, as a first step.

Reply:
done. but I think I will soon move it back as a method :)